src/Telemetry/Library/InternalsVisibleToTests.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Microsoft.Quantum.Telemetry.Tests")]
```

Comment:
```suggestion
[assembly: InternalsVisibleTo("Microsoft.Quantum.Telemetry.Tests" + SigningConstants.PUBLIC_KEY)]
```
needs to change for signing.

Reply:
I removed this attribute.